src/database/migrations/1706692679617-UserFollowMigration.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+  Table,
+  TableForeignKey,
+} from "typeorm";
+import { TableDB } from "../../common/enums/table-db.enum";
+
+export class UserFollowMigration1706692679617 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: TableDB.USER_FOLLOW,
+        columns: [
+          {
+            name: "id",
+            type: "uuid",
+            isPrimary: true,
+            generationStrategy: "uuid",
+            default: `uuid_generate_v4()`,
```

Comment:
should have a migration helper file which has some helper for id generation, timestamp to reuse b.c they are used in many where